### PR TITLE
Add multiple checkboxes form questions for meeting times and meeting facilitator

### PR DESCRIPTION
**Pull request by viviancan**
Refs #4698 

This is the final PR for the main work for the chapter program information form. This PR adds the multiple checkboxes form questions for meeting times and meeting facilitator

### CODE REVIEW

File: db/migrate/20240513150037_create_chapter_program_information_meeting_times.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateChapterProgramInformationMeetingTimes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :chapter_program_information_meeting_times do |t|
+      t.references :chapter_program_information, foreign_key: {to_table: :chapter_program_information}, index: {name: "index_cpi_mt_on_chapter_program_information_id"}
```
Comment (viviancan) [inline db/migrate/20240513150037_create_chapter_program_information_meeting_times.rb:4]: Originally when running this migration I received this error:
```
ArgumentError: Index name ‘index_chapter_program_information_meeting_times_on_chapter_program_information_id’ on table ‘chapter_program_information_meeting_times’ is too long; the limit is 63 characters
```
So I added a custom index name, but then encountered this error
```
Caused by:
PG::DuplicateTable: ERROR:  relation “chapter_program_information_id” already exists
```
So I decided to go with a custom abbreviated index name

File: db/migrate/20240513182351_create_meeting_facilitators.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateMeetingFacilitators < ActiveRecord::Migration[6.1]
+  def change
+    create_table :meeting_facilitators do |t|
+      t.string :name
+      t.integer :order
```
Comment (shaun-technovation) [inline db/migrate/20240513182351_create_meeting_facilitators.rb:5]: I think I mentioned this in another PR, but it doesn't look like `order` is being used anywhere? 🤔

File: app/views/chapter_ambassador/chapter_program_information/_form.en.html.erb [hunk 3]
```diff
@@ -74,6 +74,14 @@
                 } %>
   </div>
 
+  <div class="mb-8">
+    <%= f.association :meeting_times,
+                      as: :check_boxes,
+                      label_method: :time,
+                      label: t("chapters.program_information.meeting_time_question") %>
```
Comment (shaun-technovation) [inline app/views/chapter_ambassador/chapter_program_information/_form.en.html.erb:81]: The other thing I'm noticing and this is definitely not a blocker is the inconsistency w/ naming, here we reference `chapters.program_information`, but the model reference is `chapter.chapter_program_information`.

**Review by shaun-technovation — APPROVED** [review]
Left a couple of comments, but no blockers, LGTM! 👍

File: db/migrate/20240513182351_create_meeting_facilitators.rb (hunk 2 above)
Comment (viviancan) [reply db/migrate/20240513182351_create_meeting_facilitators.rb]: I removed order!

File: app/views/chapter_ambassador/chapter_program_information/_form.en.html.erb (hunk 3 above)
Comment (viviancan) [reply app/views/chapter_ambassador/chapter_program_information/_form.en.html.erb]: Updated this 👍

**Pull request merged**